src/table/virtualized-table/Wrapper.tsx

```diff
@@ -2,18 +2,18 @@ import React, { useMemo, useState } from 'react';
 import { WrapperProps } from './types';
 import PaginationContent from '../components/footer/PaginationContent';
 import { StyledTableWrapper } from '../components/styles';
-import { PageInfo, TableData } from '../../types';
+import { PageInfo } from '../../types';
 import FooterWrapper from '../components/footer/FooterWrapper';
 import Table from './Table';
 import { MAX_PAGE_SIZE } from './constants';
 import useOnPropsChange from './hooks/use-on-props-change';
 import { TableContext, useContextSelector } from '../context';
 
 export default function Wrapper(props: WrapperProps) {
-  const { rect } = props;
-  const { layout, theme, constraints } = useContextSelector(TableContext, (value) => value.baseProps);
-  const totalRowCount = layout.qHyperCube.qSize.qcy;
+  const { rect, tableData } = props;
+  const { paginationNeeded, totalRowCount } = tableData;
```

Comment:
same here for `tableData`, but i think you probably don need them if you use `tableData.totalRowCount` and `tableData.paginationNeeded`. unless variable name is important in this case

Reply:
It the same story here. Prettier will turn 2 easly readable lines of code into 3 lines of code. Which IMO does not add any value.

Also I think this is just my personal preference that I prefer to write `paginationNeeded` over `tableData.paginationNeeded` as it's shorter